images/macos/software-report/SoftwareReport.Toolcache.psm1

```diff
@@ -28,9 +28,22 @@ function Get-ToolcacheNodeVersions {
     return Get-ChildItem $toolcachePath -Name | Sort-Object { [Version]$_ }
 }
 
-function Get-ToolcacheGoVersions {
-    $toolcachePath = Join-Path $env:HOME "hostedtoolcache" "Go"
-    return Get-ChildItem $toolcachePath -Name | Sort-Object { [Version]$_ }
+function Get-ToolcacheGoTable
```

Comment:
Use the same code style like all functions in this file:
```
function name {
}

foreach {
}
```

Reply:
Fixed it